Organiser.Core/Organiser.Core/Controllers/BugsController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Organiser.Core.Models.ViewModels.BugsViewModels;
+using Organiser.Cores.Context;
+using Organiser.Cores.Entities;
+using Organiser.Cores.Models.Enums;
+using Organiser.Cores.Services;
+using System;
+
+namespace Organiser.Cores.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    public class BugsController : ControllerBase
```

Comment:
Controller in general, based on SRP (SOLID) is meant to be responsible for only one thing - it is an end middleware of the http request pipeline and it's job is it deserialize and bind the incoming request and pass it further. I'd avoid using bare db context right here because it violates many principles - among many others, it exposes the implementation. what if you'd like to change the database to some other technology? you can't do it now without refactoring the project. I'd suggest to use an interface and hide implementation of DB access (at least)